components/footer/_footer.scss

```diff
@@ -34,3 +34,18 @@
     }
   }
 }
+.footer--eq {
+    .footer__link {
+      float: left;
```

Comment:
This should be a modifier or a utility class on the `<li>` 
Either `class="footer__list--inline"` or `class="u-inline"`